_data/finance.yml

```diff
@@ -26,7 +26,9 @@ websites:
       url: https://www.freeagent.com
       img: freeagent.png
       twitter: freeagent
```

Comment:
Hello @danaeshell, 😄 
The Twitter tag should be removed when the `tfa` tag is changed to `Yes`. Please remove it and the build should pass 😉